Makefile

```diff
@@ -146,7 +146,7 @@ $(BUILT_JS_ASSETS): $(NODE_MODULES)
 	echo 'window.GUI_VERSION = {"version": "$(CURRENT_VERSION)", "commit": "$(CURRENT_COMMIT)"};' > $(GUIBUILD)/app/assets/javascripts/version.js
 	find $(BUILT_JS_ASSETS) -type f -name "*.js" \
 		-not -name "react*" \
-		-not -name "js-macaroon*" \
+		-not -name "js-macaroon*" | \
```

Comment:
driveby fix, these commands can likely be revisited in the future as we only require it for a single asset now.